app/src/main/java/com/github/steroidteam/todolist/broadcast/ReminderLocationBroadcast.java

```diff
@@ -18,22 +18,30 @@
 import androidx.core.app.NotificationManagerCompat;
 import com.github.steroidteam.todolist.R;
 import com.github.steroidteam.todolist.view.ItemViewFragment;
+import com.google.android.gms.location.Geofence;
+import com.google.android.gms.location.GeofencingClient;
+import com.google.android.gms.location.GeofencingRequest;
+import com.google.android.gms.location.LocationServices;
 
 public class ReminderLocationBroadcast extends BroadcastReceiver {
     public static int REMINDER_LOC_ID = 201;
     public static String REMINDER_LOC_CHANNEL_ID = "notifyLocationChannelID";
     public static CharSequence NAME_LOCATION_CHANNEL = "ReminderLocationChannel";
     public static String DESCRIPTION_LOCATION_CHANNEL = "Channel for the location reminder";
     // The radius of the central point alert region in meters
-    private static final float RADIUS = 100;
+    private static final float RADIUS = 1000;
+    private static int unique_ID = 0;
+    private static GeofencingClient geofencingClient;
 
     @Override
     public void onReceive(Context context, Intent intent) {
+        String taskDescription =
+                intent.getStringExtra(context.getString(R.string.content_location_reminder));
         NotificationCompat.Builder builder =
                 new NotificationCompat.Builder(context, REMINDER_LOC_CHANNEL_ID)
                         .setContentTitle(context.getString(R.string.title_location_reminder))
                         .setSmallIcon(R.drawable.ic_notification)
-                        .setContentText(context.getString(R.string.content_location_reminder))
+                        .setContentText(taskDescription)
```

Comment:
Quick question: is this the content of the notification? I am a bit confused because I don't fully get the purpose of the entry in `strings.xml` :thinking:.